db/cryptodata/config.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cryptodata
+
+import (
+	"crypto/rsa"
+	"io/ioutil"
+	"encoding/pem"
+	"crypto/x509"
+	"errors"
+)
+
+// Config is used to read private key from file
+type Config struct {
+	// Private key file is used to create rsa.PrivateKey from this PEM path
+	PrivateKeyFiles []string `json:"private-key-files"`
+}
+
+// ClientConfig is result of converting Config.PrivateKeyFile to PrivateKey
+type ClientConfig struct {
+	// Private key is used to decrypt encrypted keys while reading them from store
+	PrivateKeys []*rsa.PrivateKey
+}
+
+// ReadCryptoConfig reads private key from PEM path
+func ReadCryptoConfig(config Config, clientConfig *ClientConfig) (error) {
+	if len(config.PrivateKeyFiles) == 0 {
+		return nil
+	}
+
+	for _, file := range config.PrivateKeyFiles {
+		bytes, err := ioutil.ReadFile(file)
+		if err != nil {
+			return err
+		}
+
+		block, _ := pem.Decode(bytes)
```

Comment:
Reading the docs for `pem.Decode` it seems to me that it can support multiple keys, by returning the rest of file after first key.

So, perhaps loop can be used here and after successful decode, when `p != nil`, it should try to decode the rest in next iteration until we get `p == nil`.